infra/postdeployment.tf

```diff
@@ -91,3 +91,50 @@ curl ${local.server_url}/api/products/?warmup
 shutdown -h now
 EOT
 }
+
+
+resource "google_compute_instance" "placeholder_init" {
+  count = var.init ? 1 : 0
+
+  depends_on = [
+    google_project_service.enabled,
+    google_cloud_run_v2_job.placeholder,
+  ]
+
+  name           = var.random_suffix ? "placeholder-initialize-${random_id.suffix.hex}" : "placeholder-initialize"
+  machine_type   = "n1-standard-1"
+  zone           = var.zone
+  desired_status = "RUNNING"
+
+  allow_stopping_for_update = true
+
+  boot_disk {
+    initialize_params {
+      image = "debian-cloud/debian-11"
+    }
+  }
+
+  network_interface {
+    network    = google_compute_network.gce_init[0].self_link
+    subnetwork = google_compute_subnetwork.gce_init[0].self_link
+
+    access_config {
+      // Ephemeral public IP
+    }
+  }
+
+  service_account {
+    email  = google_service_account.compute[0].email
+    scopes = ["cloud-platform"] # TODO: Restrict??
+  }
+
+  metadata_startup_script = <<EOT
+#!/bin/bash
+
+echo "Running placeholder deployment"
+gcloud beta run jobs execute ${google_cloud_run_v2_job.placeholder.name} --wait --project ${var.project_id} --region ${var.region}
+curl -X PURGE "${local.firebase_url}/"
```

Comment:
discussion: This cache purge shouldn't be necessary because we don't expect the firebase site URL to be exposed and checked before it's deploy is complete, unlike the final deploy which ends after tf apply completes. Would be great to learn if that's wrong (and if so, could we have an inline comment?)

Reply:
When I was testing this out, I had the URL open in a window, with the "Site not configured", and I found this helped clean that out with the placeholder. It shouldn't *have* to happen, I just found more success keeping it in.